docker/Dockerfile.specific_proxy_service

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+       python3-pip python3-setuptools python3-dev \
```

Comment:
It is a convention to put one dependency per line.

Reply:
Splitted and sorted.

Reply:
They are not sorted :)

Reply:
Almost, now it should be OK :).